docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Event subscribers registration
+description: Event subscribers registration extension scenario
+last_updated: Mar 13, 2023
+template: concept-topic-template
+related:
+- title: Plugins registration
+  link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.html
+- title: Event subscribers registration
+  link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.html
+- title: Modules configuration
+  link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/modules-configuration.html
+---
+
+## Introduction
+
+Manifests support event subscribers registration in the dependency provider.
```

Comment:
```suggestion
Manifests support event subscribers registration in the dependency provider.
```
Editor's note: What does 'in the dependency provider' mean? What's a dependency provider in this context?

Reply:
It's some type of code class, and we support registration inside them and don't support it in another class